<ui version="4.0" >
 <class>SegmentationBase</class>
 <widget class="Q3MainWindow" name="SegmentationBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Demo App for Segmentation</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>27</y>
     <width>1004</width>
     <height>930</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QLabel" name="mp_labelOrig" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Original Image/Trimap (use mousewheel)</string>
        </property>
        <property name="pixmap" >
         <pixmap>image0</pixmap>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mp_labelSegmented" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Segmented Image</string>
        </property>
        <property name="pixmap" >
         <pixmap>image0</pixmap>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line1_2" >
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Options</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QComboBox" name="mp_comboAlgo" >
         <item>
          <property name="text" >
           <string>Kolmogorov</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>FreeImpl</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Boost_PushRelabel</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Boost_Edmund_Karp</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Boost_Kolmogorov</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_buttonSegment" >
         <property name="text" >
          <string>Segme&amp;nt</string>
         </property>
         <property name="shortcut" >
          <string>Alt+N</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="1" column="0" >
          <widget class="QLabel" name="textLabel1_2_2" >
           <property name="text" >
            <string>Neighborhood Weight:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="mp_spinSimilarityWeight" >
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="textLabel1_2" >
           <property name="text" >
            <string>Similarity Weight:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="mp_spinNeighborhoodWeight" >
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>10000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Neighborhoud-Functor Threshold:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mp_spinHuePenalty" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Q3TextEdit" name="mp_logEdit" >
      <property name="maximumSize" >
       <size>
        <width>32767</width>
        <height>100</height>
       </size>
      </property>
      <property name="textFormat" >
       <enum>Qt::LogText</enum>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenImageAction" />
    <addaction name="fileOpenMaskAction" />
    <addaction name="fileNewMaskAction" />
    <addaction name="fileSaveAsActionss" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="Options" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="optionsStore_problem_as_DIMACSAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction" />
   </widget>
   <addaction name="File" />
   <addaction name="Options" />
   <addaction name="helpMenu" />
  </widget>
  <action name="fileNewAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image1</normaloff>image1</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="iconText" >
    <string>New</string>
   </property>
   <property name="shortcut" >
    <string>N</string>
   </property>
   <property name="name" stdset="0" >
    <string>fileNewAction</string>
   </property>
  </action>
  <action name="fileOpenImageAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image2</normaloff>image2</iconset>
   </property>
   <property name="text" >
    <string>&amp;OpenImage...</string>
   </property>
   <property name="iconText" >
    <string>OpenImage</string>
   </property>
   <property name="toolTip" >
    <string>OpenImage</string>
   </property>
   <property name="shortcut" >
    <string>O</string>
   </property>
   <property name="name" stdset="0" >
    <string>fileOpenImageAction</string>
   </property>
  </action>
  <action name="fileSaveAsActionss" >
   <property name="text" >
    <string>Save Image &amp;As...</string>
   </property>
   <property name="iconText" >
    <string>Save Image &amp;As...</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
   <property name="name" stdset="0" >
    <string>fileSaveAsActionss</string>
   </property>
  </action>
  <action name="filePrintAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image3</normaloff>image3</iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="shortcut" >
    <string>P</string>
   </property>
   <property name="name" stdset="0" >
    <string>filePrintAction</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
   <property name="name" stdset="0" >
    <string>fileExitAction</string>
   </property>
  </action>
  <action name="editUndoAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image4</normaloff>image4</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="iconText" >
    <string>Undo</string>
   </property>
   <property name="shortcut" >
    <string>Z</string>
   </property>
   <property name="name" stdset="0" >
    <string>editUndoAction</string>
   </property>
  </action>
  <action name="editRedoAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image5</normaloff>image5</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="iconText" >
    <string>Redo</string>
   </property>
   <property name="shortcut" >
    <string>Y</string>
   </property>
   <property name="name" stdset="0" >
    <string>editRedoAction</string>
   </property>
  </action>
  <action name="editCutAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image6</normaloff>image6</iconset>
   </property>
   <property name="text" >
    <string>&amp;Cut</string>
   </property>
   <property name="iconText" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>X</string>
   </property>
   <property name="name" stdset="0" >
    <string>editCutAction</string>
   </property>
  </action>
  <action name="editCopyAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image7</normaloff>image7</iconset>
   </property>
   <property name="text" >
    <string>C&amp;opy</string>
   </property>
   <property name="iconText" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>C</string>
   </property>
   <property name="name" stdset="0" >
    <string>editCopyAction</string>
   </property>
  </action>
  <action name="editPasteAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image8</normaloff>image8</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="iconText" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>V</string>
   </property>
   <property name="name" stdset="0" >
    <string>editPasteAction</string>
   </property>
  </action>
  <action name="editFindAction" >
   <property name="icon" >
    <iconset>
     <normaloff>image9</normaloff>image9</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find...</string>
   </property>
   <property name="iconText" >
    <string>Find</string>
   </property>
   <property name="shortcut" >
    <string>F</string>
   </property>
   <property name="name" stdset="0" >
    <string>editFindAction</string>
   </property>
  </action>
  <action name="helpContentsAction" >
   <property name="text" >
    <string>&amp;Contents...</string>
   </property>
   <property name="iconText" >
    <string>Contents</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
   <property name="name" stdset="0" >
    <string>helpContentsAction</string>
   </property>
  </action>
  <action name="helpIndexAction" >
   <property name="text" >
    <string>&amp;Index...</string>
   </property>
   <property name="iconText" >
    <string>Index</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
   <property name="name" stdset="0" >
    <string>helpIndexAction</string>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
   <property name="name" stdset="0" >
    <string>helpAboutAction</string>
   </property>
  </action>
  <action name="fileOpenMaskAction" >
   <property name="text" >
    <string>OpenMask...</string>
   </property>
   <property name="iconText" >
    <string>OpenMask</string>
   </property>
   <property name="name" stdset="0" >
    <string>fileOpenMaskAction</string>
   </property>
  </action>
  <action name="fileNewMaskAction" >
   <property name="text" >
    <string>NewMask</string>
   </property>
   <property name="iconText" >
    <string>NewMask</string>
   </property>
   <property name="name" stdset="0" >
    <string>fileNewMaskAction</string>
   </property>
  </action>
  <action name="optionsStore_problem_as_DIMACSAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Store problem as DIMACS</string>
   </property>
   <property name="iconText" >
    <string>Store problem as DIMACS</string>
   </property>
   <property name="name" stdset="0" >
    <string>optionsStore_problem_as_DIMACSAction</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
